conda_gitenv/resolve.py

```diff
@@ -60,13 +79,18 @@ def build_manifest_branches(repo):
             warnings.warn('Branch {} cannot be resolved as it contains at '
                           'least one "-" character.'.format(name))
             continue
+        if envs is None:
+            envs = ['*']
```

Comment:
It may make more sense to move this out of the `for branch in repo.branches:` loop.
Otherwise, if you have a lot of branches, you end up doing this check a lot.